recovery_points_for_virtual_server.rb

```diff
@@ -77,4 +80,16 @@ def get_vm_recovery_points(resource_id)
     vm_points.is_a?(Hash) ? [vm_points] : vm_points
   end
 
+  def get_backup_size(backup_id)
+    api_get(
+      build_query(:BackupFile, BackupUid: identifier_to_uid(:Backup, backup_id))
+    )
+    .dig(:QueryResult, :Entities, :BackupFiles, :BackupFile)
+    .map(&(proc { |hash| hash.transform_keys(&:downcase) }))
```

Comment:
not sure I understand the trick with `proc`. Why not

```ruby
.map { |hash| hash.transform_keys(&:downcase) }
```
?

Reply:
Yeah, to complicated
Your approach is more clear